hack/make-rules/test-e2e-node.sh

```diff
@@ -151,6 +155,7 @@ if [ "${remote}" = true ] ; then
     --delete-instances="${delete_instances}" --test_args="${test_args}" --instance-metadata="${metadata}" \
     --image-config-file="${image_config_file}" --system-spec-name="${system_spec_name}" \
     --preemptible-instances="${preemptible_instances}" --extra-envs="${extra_envs}" --test-suite="${test_suite}" \
+    "${timeout_arg}" \
```

Comment:
what does this do if empty? is an empty arg tolerated?

Reply:
Ahh, yeah, that works fine! Running with `TIMEOUT=20s`, `TIMEOUT=""` and without `TIMEOUT` works.

Reply:
ping @liggitt